public/style.css

```diff
@@ -195,13 +202,14 @@ img.emoji {
   width: 1.25rem;
 }
 
-td.icon-column {
+td.icon-column,
+td.emoji-column {
   margin: 0;
-  padding: 0 !important;
+  padding: 0 0.2rem !important;
   text-align: center;
 }
 
-td.location {
+td.breakout {
```

Comment:
I changed this from "breakout" to "location", seems like this change is undoing that, please fix and confirm it looks how you want now that it's locations

Reply:
I had a feeling I'd missed one - was doing most of this last night before locations was baked in. Should be correct now.